src/main/native/pairhmm/IntelPairHmm.cc

```diff
@@ -37,7 +37,22 @@ JNIEXPORT void JNICALL Java_com_intel_gkl_pairhmm_IntelPairHmm_initNative
   g_use_double = use_double;
   
 #ifdef _OPENMP
-  g_max_threads = std::min((int)max_threads, omp_get_max_threads());
+  int avail_threads = omp_get_max_threads();
+  int req_threads = (int)max_threads;
```

Comment:
Do you need the typecast on max_threads? We needed it in `std::min`, not sure if we need it here.